src/cases/containers/case-list/case-list.component.ts

```diff
@@ -234,6 +237,8 @@ export class CaseListComponent implements OnInit, OnDestroy {
       this.onPaginationSubscribeHandler(paginationDataFromResult);
     }
 
+    if (typeof resultView.results !== 'undefined') this.resultViewIsReady = true;
```

Comment:
This will only ever be evaluated when the containing method is first called. Is that the desired behaviour or should it go back to false if `resultView.results` is `undefined` at a later point?

Reply:
Same for the one below...

Reply:
This is desired. The issue is that the resultView was being used in the template as an if, defaulted to null. But with the ngrx store, when the component first loads it gets an empty object  back. This being a truth value displayed the ccd-search result before the object had been properly hydrated. The effect of this was that the search-result component believed that a search had occurred and there were no results, so shows the "No cases found..." message. 

So what I have done here is to make sure that the search-result does not show until the object has been hydrated into a proper SearchResultView object